src/AddItem/AddItem.tsx

```diff
@@ -1,8 +1,13 @@
-import { useState } from 'react';
+import { FC, useState } from 'react';
 import s from 'AddItem//index.module.css'
-import { AddItemProps } from 'types';
+import { Element} from 'types'
```

Comment:
a bit more explicit and clear
```suggestion
import type { Element } from 'types'
```